sdk/compatibility/src/lib.rs

```diff
@@ -236,6 +239,45 @@ fn get_events() -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[ignore]
+fn get_events_via_websocket_stream() -> Result<()> {
+    let env = Environment::from_env();
+    let client = env.client();
+
+    let rt = Runtime::new().unwrap();
+    let ws_client = rt
+        .block_on(env.websocket_client())
+        .unwrap_or_else(|e| panic!("Error connecting to WS endpoint: {}", e));
+
+    let (mut s_client, _) = rt.block_on(StreamingClient::new(ws_client, 10));
```

Comment:
This doesn't seem very ergonomic, requiring the user to first make the websocket and then the streaming client. I think this should probably be handled transparently and in one step (especially since we only have a websocket implementation at the moment)